### ISSUE TITLE
⛔ 🐞 Bug Report: aria-*] attributes do not match their roles - popup

### ISSUE THREAD

**Issue by LydiaDA** [elementor/elementor#22555]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I have used a template from the library to create a cookie popup.

Google PSI gives a notification: aria-*] attributes do not match their roles.

To fix an aria issue, Elementor gives this information: https://elementor.com/blog/wordpress-accessibility-elementor/ 

So following these instructions, I added this line to the container widget in advanced settings - attributes:

aria-label|cookies consent

And I tried: aria-label|"cookiesconsent" - after Elementor Support gave a different way of putting the value in.

But it doesn't solve the issue. It is still flagged by Google Page Speed Insights as role error.

Could you please look into this?

Thanks, 

Lydia

### Steps to reproduce

Check the popups in the library (using flex containers). I used one, but that was in the beginning of this year - I think the library has changed since then. 
Check them on a testing site whether Google PSI gives this notification for the popup:  aria-*] attributes do not match their roles 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.35
	PHP Version: 8.1.19
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://amorpha.nl
	Home URL: https://amorpha.nl
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: nl
	Timezone: 1
	Admin Email: [REDACTED]
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: nl_NL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/113.0

== Active Plugins ==
	Elementor
		Version: 3.13.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.2
		Author: Elementor.com

	FlyingPress
		Version: 4.5.0
		Author: 

	MailerLite - Signup forms (official)
		Version: 1.6.7
		Author: MailerLite

	Rank Math SEO
		Version: 1.0.115
		Author: Rank Math

	WPvivid Backup Plugin
		Version: 0.9.86
		Author: WPvivid Team


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com


== Eigenschappen ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


````

</details>

**Comment by nicholaszein** [comment]
Hello @LydiaDA,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
You can disregard the warning. This is a false positive. 

All Elementor Popups have an `aria` attribute of `aria-modal="true"` which informs the assistive technology user (amongst other things) that the windows underneath the current dialog  (or in this case the Elementor Popup) are not part of the modal content.

### ✅ What you can do to solve your issue:
Please take the Page Speed Insights reports as what they are, insights, or suggestions to improve your website. They are not to be taken as literal issues that need to be solved even if it means undermining the user experience. This role is important to ensure AT users can understand what it means.

If you wish to improve the experience you can do two things:

1. Enable the option for "Accessible Navigation" in the Popup Settings > Advanced tab.
2. Enable the option to show the overlay of the Popup as well as prevent the user from navigating outside of it.

This is however not a fix, it will trick Google's PSI to think that it is a modal. But again, that is not relevant for the AT user, since Assistive Technology users will understand it's a modal regardless.

### 👍 For now, that's all we can help with. 

### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
